metal/classifier.py

```diff
@@ -322,6 +325,10 @@ def _train_model(
                     "validation_scoring_kwargs"
                 ]
                 self.eval()
+
+                # Adding loss function to support validating on loss
```

Comment:
What is this comment referring to/why is it hanging?

Reply:
Silliness. I'll get rid of it :).